source/unsafe-messenger.js

```diff
@@ -6,12 +6,14 @@
 export default function unsafeMessenger() {
 	document.body.addEventListener('gt:get', listener);
 
+	// Add a new condition to check for the existence of the monaco object
 	function listener({target}) {
 		if (target.CodeMirror) {
 			codeMirror(target);
+		} else if (monaco) {
```

Comment:
While it might be ok, just because it exists it doesn't mean this specific field is Monaco. For example this would break the Ace field on GhostText's demo page.

Can you find a way to see if this specific field is a Monaco field like it's doing for CodeMirror?

Reply:
Done ✅